### PR TITLE
add editType 'finishMovePosition'

### PR DESCRIPTION
**Pull request by matthrice**
Add new `editType = 'finishMovePosition'` to distinguish when a drag has finished and expose it in `onEdit` callback

### CODE REVIEW

File: src/lib/layers/editable-geojson-layer.js [hunk 1]
```diff
@@ -383,6 +407,22 @@ export default class EditableGeoJsonLayer extends EditableLayer {
...
+  handleFinishMovePosition(featureIndex: number, positionIndexes: number, groundCoords: number[]) {
+    const updatedData = this.state.editableFeatureCollection
+      .replacePosition(featureIndex, positionIndexes, groundCoords)
+      .getObject();
+
+    this.props.onEdit({
+      updatedData,
+      updatedMode: this.props.mode,
+      updatedSelectedFeatureIndex: featureIndex,
+      editType: 'finishMovePosition',
```
Comment (georgios-uber) [inline src/lib/layers/editable-geojson-layer.js]: Please move all these in an Object that is exported.

Comment (matthrice) [reply src/lib/layers/editable-geojson-layer.js]: All of the `handle` functions?

File: docs/api-reference/layers/editable-geojson-layer.md [hunk 2]
```diff
@@ -113,7 +113,9 @@ The `onEdit` event is the core event provided by this layer and must be handled
 
   * `removePosition`: A position was removed.
 
-  * `addFeature`: A new feature was added. Its index is reflected in `featureIndex`.
+  * `addFeature`: A new feature was added. Its index is reflected in `featureIndex`
+
+  * `finishMovePosition`: A position finished moving (indicated by `mouseup`).
```
Comment (supersonicclay) [inline docs/api-reference/layers/editable-geojson-layer.md]: bit of a nit, but it is actually triggered by `pointerup`, not `mouseup`.

File: src/lib/layers/editable-geojson-layer.js (hunk 1 above)
Comment (georgios-uber) [reply src/lib/layers/editable-geojson-layer.js]: @supersonicclay @matthrice Yes please do it for all of them.

**Comment by georgios-uber** [comment]
@matthrice You can merge this one, don't worry about the constant.

**Pull request merged**